src/content/en/2021/media.md

```diff
@@ -729,30 +727,28 @@ Interestingly, we see a large push away from `preload` on both mobile and deskto
 
 The next most commonly used attribute is `autoplay`. This tells the browser that the video should play as soon as possible (and because of this, autoplay will actually override the preload attribute).
 
-The autoplay attribute is a boolean attribute, meaning that its presence by fault means true. So for the 190 sites that use `autoplay="false"`, we're sorry to tell you that is not going to work.
+The autoplay attribute is a Boolean attribute, meaning that its presence by fault means true. So, for the 190 sites that use `autoplay="false"`, we're sorry to tell you that is not going to work.
 
 #### `width`
 
-The width attribute is also one of the top video attributes. It tells the browser how wide the video player should be (note that height is very rarely used, since the aspect ratio of the video will decide the space with just one value.)
-
-{# TODO: I discovered recently that this is not the case! And unless you override with CSS a default aspect-ration of 2:1 will be used: whatwg/html#3090 #}
+The width attribute is also one of the top video attributes. It tells the browser how wide the video player should be. Note that height is very rarely used, since the browser can set this - but it will use a  <a hreflang="en" href ="https://github.com/whatwg/html/issues/3090">default aspect-ratio of 2:1</a> which may be incorrect if not explicity overriden with the `aspect-ratio` CSS styling.
```

Comment:
@dougsillars I added something here for this. Let me know if that's good with you, or you want something else instead (or to just remove it completely!).

Reply:
thanks i was unaware myself!